packages/ui/src/components/va-toast/VaToast.vue

```diff
@@ -112,6 +114,17 @@ export default defineComponent({
       color: textColorComputed.value,
     }))
 
+    const computedAriaLive = computed(() => {
+      const role = attrs.role
```

Comment:
Might better have it declared as prop, because it has some logic that depends on prop...

Also, you can type it as `StringWithAutocomplete<'alert' | 'alertdialog' | 'status'>`.